### PR TITLE
feat(style): new exposed form styles option

### PR DESCRIPTION
**Pull request by runrog**

### CODE REVIEW

File: styleguide/_themes/global/scss/forms.scss [hunk 1]
```diff
@@ -228,3 +228,232 @@ table {
     }
   }
 }
+
+// exposed form overrides
+$exposed-form-bg: #003482;
+
+// sass-lint:disable-block no-ids
+#block-webform {
```
Comment (WidgetsBurritos) [inline styleguide/_themes/global/scss/forms.scss]: Can we change this to: `.rs-webform-embedded`?

I was able to tweak the HTML to this: 
```
<div class="rs-webform-block rs-webform-embedded" id="block-webform-block">
```

That `.rs-webform-embedded` class is specific to this particular block and you can see it's at the same level as that `id`.

File: styleguide/derek/global-theme/forms.ejs [hunk 2]
```diff
@@ -77,6 +77,80 @@
   </form>
 </div>
 
+<div class="row half-padding">
+  <h5>Exposed Form</h5>
+  <div id="block-webform">
```
Comment (WidgetsBurritos) [inline styleguide/derek/global-theme/forms.ejs]: Change this to `<div class="rs-webform-block rs-webform-embedded" id="block-webform-block">` to match what we have in production and then target off of `rs-webform-embedded`

**Comment by runrog** [comment]
@WidgetsBurritos that error message injecting to the top of the form threw a major wrench. Now we have to target the other generic `.form-items` uglier. `.form-item:nth-type-of(num)` doesn't work as expected so I had to go with a nested sibling selector. The other option would be js. Rather not go that route but it should be working now.

**Review by WidgetsBurritos — APPROVED** [review]
👍 LGTM

**Pull request merged**